src/chapter_1_examples/hello_you/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "hello_you"
 version = "0.1.0"
-authors = ["Chris James <qui666@gmail.com>"]
-edition = "2018"
+authors = ["Cooper Gillan <cooper.gillan@gmail.com>"]
+edition = "2021"
```

Comment:
I actually did have to blow away the `hello_you/` directory and re-run `cargo init hello_you` to get this set up as needed.